tools/osv-linter/internal/helpers/utility.go

```diff
@@ -0,0 +1,70 @@
+package helpers
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/sethvargo/go-retry"
+)
+
+// Make a HTTP GET request for url and retry 3 times, with an exponential backoff.
+func Get(url string) (resp *http.Response, err error) {
+	backoff := retry.NewExponential(1 * time.Second)
+	if err := retry.Do(context.Background(), retry.WithMaxRetries(3, backoff), func(ctx context.Context) error {
+		req, err := http.NewRequest("GET", url, nil)
+		if err != nil {
+			return err
+		}
+
+		r, err := http.DefaultClient.Do(req)
+		if err != nil {
+			return err
+		}
+		// defer r.Body.Close()
```

Comment:
why is this commented out?

Reply:
> why is this commented out?

I've removed it altogether in c9090ae. Because this function returns the HTTP response in its entirety, closing the body here is undesired. It's coupled with the calls to `faulttolerant.Get` in `pkgchecker`.